### DISCUSSION TITLE
how to change ui [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2091] [Discord Threads]
> 🙋‍♂️ **李思南**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### can u be more specific? which part u wanna change?  
#  
> 🙋‍♂️ **李思南**  
#### I want to run in local ,and  
![image0](https://cdn.discordapp.com/attachments/973990574086566010/974211973283733504/Snipaste_2022-05-12_15-29-59.png)  
#  
> 🙋‍♂️ **李思南**  
#### cant not start backed  
#  
> 🙋‍♂️ **李思南**  
#### I thought it was a system problem, but linux is the same as window  
#  
> 🙋‍♂️ **李思南**  
#### If I can't run it locally, I can't localize the ui  
#  
> 👨‍🚀 **wingkwong**  
#### have u started frontend?  
#  
> 👨‍🚀 **wingkwong**  
#### for backend, u may check if the port s being used  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973990574086566010)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
